app/Http/Controllers/PublicController.php

```diff
@@ -19,7 +19,14 @@ public function about()
     {
         return view('about');
     }
-
+    public function four0four()
```

Comment:
no need for this also

Reply:
done

Reply:
it was for testing in static pages  , thanks dear